src/components/LoadingOverlay/__stories__/LoadingOverlay.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import type {Meta, StoryFn} from '@storybook/react';
+
+import {Button} from '../../Button';
+import {spacing} from '../../layout';
+import {block} from '../../utils/cn';
+import {LoadingOverlay} from '../LoadingOverlay';
+import type {LoadingOverlayProps} from '../LoadingOverlay';
+
+import './LoadingOverlay.scss';
+
+const b = block('loading-overlay-showcase');
+
+export default {
+    title: 'Components/Feedback/LoadingOverlay',
+    component: LoadingOverlay,
+} as Meta;
+
+const DefaultTemplate: StoryFn<LoadingOverlayProps> = (args) => {
+    const [showContent, setShowContent] = React.useState<boolean>(true);
+
+    return (
+        <div className={b()}>
+            <LoadingOverlay {...args} contentClassName={b('content')} minHeight={44}>
+                {showContent && (
+                    <React.Fragment>
+                        <div>I am an example</div>
+                        <div>content</div>
+                    </React.Fragment>
+                )}
+            </LoadingOverlay>
+            <Button className={spacing({mt: 2})} onClick={() => setShowContent((value) => !value)}>
+                Toggle content
+            </Button>
+        </div>
+    );
+};
+export const Default = DefaultTemplate.bind({});
```

Comment:
pls add more examples in storybook.
I think overlay in dialog content and table overlay would be enough.
It is better to do an example with a table when there is no data at first and they are loaded worse to show how overlay min height work